descriptors/ModuleDescriptor-template.json

```diff
@@ -1,11 +1,95 @@
 {
-  "id": "${artifactId}-${version}",
-  "name": "",
+  "id": "@artifactId@-@version@",
+  "name": "Remote storages API module",
   "requires": [],
-  "provides": [],
-  "permissionSets": [],
+  "provides": [
+    {
+      "id": "remote-storage-configurations",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["POST"],
+          "pathPattern": "/remote-storages/configurations",
```

Comment:
Is this RESTfull to use name remote-storages in plural? I think this is not a resource retrieved by id.  CC @MikhailFokanov

Reply:
@MikhailFokanov, do you have a chance to review this? I don't like idea that your reply absence can be blocker for this PR.